templates/contribute/tags_and_branches.md

```diff
@@ -58,23 +66,36 @@ Mathlib.
   "Mathlib reviewers" stream.
 * Success in CI on the `nightly-testing` branch results in the creation or updating of a branch
   `nightly-testing-YYYY-MM-DD` to match that commit.
-  * Thus if `nightly-testing-YYYY-MM-DD` exists, we know that:
+  * Thus if `nightly-testing-YYYY-MM-DD` exists, we know that on it:
     * the `lean-toolchain` is `leanprover/lean4:nightly-YYYY-MM-DD`, and
     * CI succeeds.
 * When changes are required to Std to adapt to a breaking change in Lean,
-  please open a PR.
-  * Typically, if the change was made in `leanprover/lean4#NNNN`,
-    this PR should be from a branch named `lean-pr-testing-NNNN`,
-    and `lean-toolchain` will contain `leanprover/lean4-pr-releases:pr-release-NNNN`.
-  * This PR should be made against the `bump/v4.X.0` branch, not against `main`.
-  * This PR should be labelled `v4.X.0`, where `v4.X.0` is the upcoming stable version,
-    and also labelled 'depends on core changes'.
+  you will need to make a branch, and later open a PR from that branch.
+  (Note that the steps below happen automatically at Mathlib,
+  but need to be done manually for Std.)
+  * If the change was made in `leanprover/lean4#NNNN`,
+    then this branch should be called `lean-pr-testing-NNNN`.
+  * This branch should be based off the branch `nightly-testing-YYYY-MM-DD`
+    where `YYYY-MM-DD` is the date of the nightly release that your Lean PR is based off
+    (you can check this will `script/most-recent-nightly-tag.sh` in the Lean repository).
+  * If this branch does not yet exist, you will need to wait
+    (and possibly move forward to a subsequent nightly).
+    Contact @semorrison for assistance if needed.
+  * Ideally you will push this branch to the main Std repository; we can provide write access if needed.
+  * The `lean-toolchain` on this branch must contain `leanprover/lean4-pr-releases:pr-release-NNNN`.
+  * You may open a PR from the `lean-pr-testing-NNNN` branch, either before or after
+    making the required adaptations.
+  * When opening the PR, remember to set the base branch to `nightly-testing-YYYY-MM-DD`.
+  * Note that the PR, when approved, will not actually be merged into `nightly-testing-YYYY-MM-DD`
```

Comment:
Is this important for the person submitting the PR? I.e. is there any action where this note would influence what they should do?

Reply:
I've removed this, and made sure this is clearly explained when talking about what a maintainer will do.